app/atoms/ln-patternlab/_ln-m-widget-contact.scss

```diff
@@ -0,0 +1,26 @@
+$_phone_prefix: 'T';
```

Comment:
Well it could have template files in the future, if we allowed you to override the patternlab templates. I'm open to what other people think? @mitogh @camilodelvasto